datafusion/catalog/src/information_schema.rs

```diff
@@ -405,7 +405,7 @@ fn get_udf_args_and_return_types(
     udf: &Arc<ScalarUDF>,
 ) -> Result<Vec<(Vec<String>, Option<String>)>> {
     let signature = udf.signature();
-    let arg_types = signature.type_signature.get_possible_types();
+    let arg_types = signature.type_signature.get_example_types();
```

Comment:
renamed and deprecated `get_possible_types` to reflect what the function actually does